CONTRIBUTING.md

```diff
@@ -33,7 +33,7 @@ Head over to [here](https://hacktoberfest.digitalocean.com/sign_up/register) to
  - [`jest-matchers-utils`](https://github.com/facebook/jest/tree/master/packages/jest-matcher-utils) is being used for syntax highlighting of error messages.
    * See the Jest docs for an [example usage](https://facebook.github.io/jest/docs/en/expect.html#thisutils)
  - Jest's [`expect`](https://github.com/facebook/jest/tree/master/packages/expect) package is being used to access their deep `equals` function.
-   * `import { equals } from 'expect/build/jasmine_utils';`
+   * `import { equals } from 'expect/build/jasmineUtils';`
```

Comment:
why? it's injected as `this.equals` in custom matchers - please don't reach into private files

https://jestjs.io/docs/en/expect#thisequalsa-b

Reply:
I just came here from the release, I know it's not really related to this PR (although this change is a good example of why it's a bad idea)